src/components/posts/PostStats.tsx

```diff
@@ -1,34 +1,28 @@
 import React, { useState } from 'react'
-import { Option } from '@polkadot/types'
-import { PostId, Post } from '@subsocial/types/substrate/interfaces/subsocial'
 import { MutedSpan } from '../utils/MutedText'
 import { PostVoters, ActiveVoters } from '../voting/ListVoters'
 import { Pluralize } from '../utils/Plularize'
 import BN from 'bn.js'
-import { withCalls, withMulti, postsQueryToProp } from '../substrate'
 import { nonEmptyStr } from '@subsocial/utils'
+import { idToBn, PostStruct } from 'src/types'
 
 type StatsProps = {
-  id: PostId
-  postById?: Option<Post>
+  post: PostStruct
   goToCommentsId?: string
 };
 
-const InnerStatsPanel = (props: StatsProps) => {
-  const { postById, goToCommentsId } = props
+export const StatsPanel = (props: StatsProps) => {
+  const { post, goToCommentsId } = props
 
   const [ commentsSection, setCommentsSection ] = useState(false)
   const [ postVotersOpen, setPostVotersOpen ] = useState(false)
 
-  if (!postById || postById.isNone) return null
-  const post = postById.unwrap()
-
-  const { upvotes_count, downvotes_count, replies_count, shares_count, score, id } = post
-  const reactionsCount = new BN(upvotes_count).add(new BN(downvotes_count))
+  const { upvotesCount, downvotesCount, repliesCount, sharesCount, score, id } = post
+  const reactionsCount = new BN(upvotesCount).add(new BN(downvotesCount))
   const showReactionsModal = () => reactionsCount && setPostVotersOpen(true)
```

Comment:
Looks like that `reactionsCount && ` check is wrong here. Because it will always result in `true`, because `reactionsCount` is a BN object - that is true.